railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -6,10 +6,7 @@ Rails.application.configure do
   # Code is not reloaded between requests.
   config.enable_reloading = false
 
-  # Eager load code on boot. This eager loads most of Rails and
-  # your application in memory, allowing both threaded web servers
-  # and those relying on copy on write to perform better.
-  # Rake tasks automatically ignore this option for performance.
+  # Eager load code on boot for copy-on-write memory savings (ignored by Rake tasks).
```

Comment:
If I put myself in the shoes of a new web developer, I would have trouble understanding copy-on-write. However I would intuitively know that eager loading upfront improves performance since you don't need to do any more loading on a per-request basis.

So the old comment feels more understandable for such a persona, but maybe it's just me. It could still be shortened however, something like this:

```suggestion
  # Eager load code on boot for for better web server performance and
  # copy-on-write memory savings (ignored by Rake tasks).
```

Reply:
Made it broader and no need to mention COW.